fairing/deployers/job/job.py

```diff
@@ -24,7 +26,7 @@ class Job(DeployerInterface):
     def __init__(self, namespace=None, runs=1, output=None,
                  cleanup=True, labels=None, job_name=constants.JOB_DEFAULT_NAME,
                  stream_log=True, deployer_type=constants.JOB_DEPLOPYER_TYPE,
-                 pod_spec_mutators=None):
+                 pvc_name=None, pvc_mount_path=None, pod_spec_mutators=None):
```

Comment:
This should be part of the backends so that we can mount pvc in all build/train/serve jobs.

Reply:
Updated to following the `kubenertes.utils.get_resource_mutator` implementation way that's great.